modules/renter/downloadcache.go

```diff
@@ -14,21 +14,33 @@ import (
 // TODO this won't be necessary anymore once we have partial downloads.
 func (udc *unfinishedDownloadChunk) addChunkToCache(data []byte) {
 	if udc.download.staticDestinationType == destinationTypeSeekStream {
-		// We only cache streaming chunks since browsers and media players tend to only request a few kib at once when streaming data. That way we can prevent scheduling the same chunk for download over and over.
+		// We only cache streaming chunks since browsers and media players tend
```

Comment:
Best to keep comments in the infinitive form.  The one exception is "TODO" comments which are typically less formal and could be in first person or the infinitive form.